### DISCUSSION TITLE
Voting for best nav & footer theme

### DISCUSSION THREAD

**Discussion by theshamkhi** [theshamkhi/GearBox_PC#2] [Polls]
Hello Team! 👋
We have two design options to choose from for our project GearBox Pc.
Feel free to share any comments or feedback to help guide the decision.

**Theme 1**: ![Capture_decran_2024-11-11_124126](https://github.com/user-attachments/assets/8d642f34-e5cd-4b90-9f59-0af68248b1b4)
**Theme 2**: ![image](https://github.com/user-attachments/assets/3a11ae21-e4a8-4000-b901-3874769fd974)

**Comment by OumaymaBrd** [comment]
I like Theme 1

**Comment by OussamaBenoujja** [comment]
since we have already intgrated the second theme i think it would be just extra hassle to go for the new one.
